site/content/integrations/prometheus.md

```diff
@@ -123,3 +123,48 @@ Also, we recommend adding `M3DB` and `M3Coordinator`/`M3Query` to your list of j
 When using the Prometheus integration with Grafana, there are two different ways you can query for your metrics. The first option is to configure Grafana to query Prometheus directly by following [these instructions.](http://docs.grafana.org/features/datasources/prometheus/)
 
 Alternatively, you can configure Grafana to read metrics directly from `M3Coordinator` in which case you will bypass Prometheus entirely and use M3's `PromQL` engine instead. To set this up, follow the same instructions from the previous step, but set the `url` to: `http://<M3_COORDINATOR_HOST_NAME>:7201`.
+
+## Using any Prometheus Remote Write capable storage
+
+M3Coordinator supports any backend that implements Prometheus Remote Write API.
```

Comment:
```suggestion
M3 Coordinator supports any backend that implements the Prometheus Remote Write API.
```

It's a bit confusing, but we're trying to use "proper" spelling for components unless referring to them as an actual code reference.